api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/ResumoMensalQueries.java

```diff
@@ -476,9 +476,9 @@ public static String getE3ExclusionCriteria(
             + " enc.encounter_datetime BETWEEN :startDate AND :endDate AND enc.encounter_type=%d AND ob.concept_id=%d) ed "
             + " ON p.patient_id=ed.patient_id "
             + " WHERE p.voided = 0 AND e.voided = 0 AND o.voided = 0 "
-            + " AND e.location_id = :location AND (e.encounter_datetime >= "
+            + " AND e.location_id = :location AND e.encounter_datetime BETWEEN "
             + " IF(MONTH(:startDate) = 12  && DAY(:startDate) = 21, :startDate, CONCAT(YEAR(:startDate)-1, '-12','-21')) "
-            + " AND <= (:startDate)-1) "
+            + " AND :startDate "
```

Comment:
This variable need to be changed to **+ " AND date_add(:startDate, interval -1 day) "**

Reply:
for it to meet this need on the ticket **Simplifying the exclusion can check >=newStartDate and <= (startDate-1d)**

Reply:
Please change in all occurrences in the exclusion

Reply:
Run package the module, install it, generate the numbers and attach results on the ticket